src/auth/constants.ts

```diff
@@ -1,9 +1,16 @@
 import 'dotenv/config';
 
-export const jwtConstants = {
-  secret: process.env.JWT_SECRET,
-};
-
 export const bcryptConstants = {
   salt: +process.env.BCRYPT_SALT_ROUNDS,
 };
+
+export const serviceConstants = { name: 'Recipe App' };
+
+export const strategyNameConstants = {
+  jwt: {
+    bearer: 'jwt.bearer',
+    pat: 'jwt.pat',
+    twoFactor: '2fa.bearer',
+  },
+  local: 'local',
+};
```

Comment:
check comment from other PR where I recommend changing the naming convention here